src/core/router/index.js

```diff
@@ -43,7 +43,6 @@ export function Router(Base) {
       this.updateRender();
       lastRoute = this.route;
 
-      // eslint-disable-next-line no-unused-vars
       router.onchange(params => {
```

Comment:
Maybe better to just have `() =>` instead of `params =>` or `_ =>`

Reply:
`params` is needed here. I have no idea why that `eslint-disable` comment was there, but like the others it's being auto-removed by `lint:fix` because it is unnecessary.